test/e2e/framework/kcp.go

```diff
@@ -95,6 +96,7 @@ func newKcpServer(t *T, cfg KcpConfig, artifactDir, dataDir string) (*kcpServer,
 			"--listen=:" + kcpListenPort,
 			"--etcd_client_port=" + etcdClientPort,
 			"--etcd_peer_port=" + etcdPeerPort,
+			"--etcd_wal_size_bytes=" + strconv.Itoa(5*1000), // 5KB
```

Comment:
I tested with `-count 100` and this seemed to work just fine, no impact on performance and no errors. Factor of 12800 less disk impact!